### PR TITLE
Minor tweaks relating to options

### PR DESCRIPTION
**Pull request by jlstevens**
So far this includes one whitespace fix and an attempt to remove a condition that appears to be unnecessary.

### CODE REVIEW

**Comment by philippjfr** [comment]
Can https://github.com/pyviz/holoviews/issues/3466 be closed?

**Comment by jlstevens** [comment]
> Can #3466 be closed?

Not yet - there is a lot more stuff mentioned there. e.g what happen to options when no backend is loaded.

**Comment by jlstevens** [comment]
@jbednar @philippjfr Ready to review/merge.

**Comment by jlstevens** [comment]
Had to restart a bunch of tests due to transients. Now everything is green except a tiny little dip in coveralls...

**Comment by philippjfr** [comment]
I'm looking at coveralls now, -0.09%!

File: holoviews/core/options.py [hunk 1]
```diff
@@ -1478,11 +1477,11 @@ def register(cls, associations, backend, style_aliases={}):
 
             opt_groups = {'plot':   Options(allowed_keywords=plot_opts),
                           'output': Options(allowed_keywords=Options._output_allowed_kws)}
-            if not isinstance(view_class, CompositeOverlay) or hasattr(plot, 'style_opts'):
-                 opt_groups.update({'style': Options(allowed_keywords=style_opts),
-                                    'norm':  Options(framewise=False, axiswise=False,
-                                                     allowed_keywords=['framewise',
-                                                                       'axiswise'])})
+
+            opt_groups.update({'style': Options(allowed_keywords=style_opts),
```
[the remark this answers is not in the document]
Comment (jlstevens) [reply holoviews/core/options.py]: Good catch. Addressed in ad66cf3.

**Comment by philippjfr** [comment]
Thanks, looks good now, merging.

**Pull request merged**